.gitmodules

```diff
@@ -1,11 +1,11 @@
 [submodule "libs/geometry.hpp"]
-	path = mapbox/geometry.hpp
+	path = deps/geometry.hpp
```

Comment:
Why changing `mapbox` to `deps`?

Reply:
To make `mapbox` as header only library. When i add just one include directory into my cmake and can include any header from it.

Reply:
Shouldn't this be `include/mapbox` instead of `mapbox` then?

Reply:
I'm not sure that i understood your question.

In proposed state we just need to add `target_include_directory(... PUBLIC mapbox-base/mapbox/include)`. Something like this.

And after that you can use these includes easy:
```cpp
#include "mapbox/std/weak.hpp"
```

Reply:
Let's take the `weak.hpp` case, that would be then placed in the following folder (according to your changes):

> mapbox-base/mapbox/include/mapbox/std/weak.hpp

Instead, I propose the use mapbox-base root directory as the include path, so we could have something like this instead:

> mapbox-base/include/mapbox/std/weak.hpp

Reply:
Also, I'd prefer to not modify the git submodule paths as cascading this on all repositories currently using `mapbox-base` will be quite challenging.

Reply:
I've moved them into separate library just to make submodule replacement process more clean. I mean process when we can move code from submodules into our header library.

Any project that use `mapbox-base` as submodule right now would not be affected. Just developers that will update submodule hash  to new one will require `git submodule sync`